src/sentry/templates/sentry/idp_email_verified.html

```diff
@@ -0,0 +1,12 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+    <h3>Thank you!</h3>
+    <p>
+        Your email has been confirmed. Click the button below to log in to your Sentry account.
+    </p>
+
+    <a href="{{ url }}" class="btn">Continue</a>
```

Comment:
we'll want this to be a form POST as to have the user be able to re-use their existing login pipeline. think you just need to wrap your button in:
```
<form class="form-stacked" action="" method="post">
    {% csrf_token %}
```

and add `type="submit"` to your button. (see https://developer.mozilla.org/en-US/docs/Learn/Forms/Sending_and_retrieving_form_data)